php/values.yaml

```diff
@@ -293,22 +294,13 @@ fpm:
   lifecycle:
     postStart: []
     # postStart: ["/bin/sh", "-c", "echo Hello > /tmp/message"]
+    preStop: []
```

Comment:
In my opinion, it would be better to set a default value for safety shutdown.

```
preStop: ["/bin/sh", "-c", "sleep 1; kill -QUIT 1; sleep {{ .Values.fpm.processControlTimeout | default 0 }}"]
```